testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/configs/ConfigHive3.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.launcher.env.configs;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.tests.product.launcher.docker.DockerFiles;
+import io.trino.tests.product.launcher.env.Environment;
+
+import javax.inject.Inject;
+
+import java.util.List;
+
+import static io.trino.tests.product.launcher.env.EnvironmentContainers.PRESTO;
+import static java.util.Objects.requireNonNull;
+import static org.testcontainers.utility.MountableFile.forHostPath;
+
+public class ConfigHive3
```

Comment:
Why do you need this config at all? Let's put as much as possible into `SinglenodeHive3`.

Reply:
Hmm I'm just trying to take over unfinished pr https://github.com/trinodb/trino/pull/6317 and there was an explicit comment saying SinglenodeHive3 should be deleted

Reply:
> Why do you need this config at all? Let's put as much as possible into `SinglenodeHive3`.

ConfigHive3 is actually the way we should do it, since we eventualy want to run more and more Hive tests on this environment.
Perhaps, we will be able to move from HDP3 to Hive3 for default testing.